src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Client.kt

```diff
@@ -57,16 +55,14 @@ data class Client(
 
   @OneToMany
   @JoinColumn(name = "registeredClientId")
-  @Where(clause = "access_token_issued_at =(select max(oa.access_token_issued_at) from oauth2_authorization oa where oa.authorization_grant_type='client_credentials')")
   val latestClientCredentialsAuthorization: MutableSet<Authorization>?,
```

Comment:
Would rename this to `latestClientAuthorization` as it could be used for authorisation code clients now too (the where clause was limiting it to client credentials).